app/bmc/src/main.c

```diff
@@ -55,29 +55,24 @@ int update_fw(void)
 	k_busy_wait(1000);
 	gpio_pin_set_dt(&reset_spi, 0);
 
-	if (IS_ENABLED(CONFIG_TT_FWUPDATE)) {
+	if (false) {
```

Comment:
The fwupdate library used `tt_boot_fs` checksums to determine if the image needed to be updated previously. However, with moving slot1 to external spi flash, mcuboot is able to determine if the images differ without any outside assistance. There is no need to have the application manually request an upgrade.